minorminer/utils/parallel_embeddings.py

```diff
@@ -181,7 +232,12 @@ def find_multiple_embeddings(
         ):
             emb = []
         else:
-            emb = embedder(_S, _T, **embedder_kwargs)
+            if perf_counter() >= timeout:
+                emb = []
```

Comment:
I think it's clearer just to have break here as well as above.
```suggestion
                break
```